pattern-library/sass/_normalize.scss

```diff
@@ -355,24 +355,12 @@ input[type="number"]::-webkit-outer-spin-button {
  */
 
 input[type="search"] {
-  -webkit-appearance: textfield; /* 1 */
   -moz-box-sizing: content-box;
   -webkit-box-sizing: content-box; /* 2 */
   box-sizing: content-box;
 }
 
 /**
- * Remove inner padding and search cancel button in Safari and Chrome on OS X.
```

Comment:
Wow, so that's why we didn't get the cancel button! I wonder why this was deliberately disabled...